plugins/package-curation-providers/clearly-defined/src/main/kotlin/ClearlyDefinedPackageCurationProvider.kt

```diff
@@ -127,11 +125,6 @@ class ClearlyDefinedPackageCurationProvider(
                     }
                 }
 
-                is JsonMappingException -> {
```

Comment:
Should we catch https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-core/kotlinx.serialization/-serialization-exception/ instead?

Reply:
I didn't see a reason to do that as this branch basically does the same as the `else` branch below.

Reply:
The error message was a bit more specific, but ok.